File: app/src/main/java/com/nativo/sampleapp/NativeAdImpl/StandardDisplayAd.java
package com.nativo.sampleapp.NativeAdImpl;

import android.content.Context;
import android.view.View;
import android.webkit.WebView;

import androidx.annotation.NonNull;
import androidx.cardview.widget.CardView;
import com.nativo.sampleapp.R;

import net.nativo.sdk.injectable.NtvStandardDisplayInjectable;

public class StandardDisplayAd implements NtvStandardDisplayInjectable {

    private CardView layout;
    private WebView webView;
    private View view;

    @NonNull
    @Override
    public WebView getContentWebView() {
        return webView;
    }

    @Override
    public void contentWebViewOnPageFinished() {

    }

    @Override
    public void contentWebViewOnReceivedError(String s) {

    }

    @Override
    public int getLayout(Context context) {
        return R.layout.standard_display;
    }

    @Override
    public void bindViews(View v) {
        view = v;
        layout = v.findViewById(R.id.standard_display_layout);
        webView = v.findViewById(R.id.standard_display_webview);
    }

    @NonNull
    @Override
    public View getView() {
        return view;
    }
}

File: app/src/main/java/com/nativo/sampleapp/NativeAdImpl/StandardDisplayAd.kt
package com.nativo.sampleapp.NativeAdImpl

import android.content.Context
import android.view.View
import android.webkit.WebView
import androidx.cardview.widget.CardView
import com.nativo.sampleapp.R
import net.nativo.sdk.injectable.NtvStandardDisplayInjectable

class StandardDisplayAd : NtvStandardDisplayInjectable {

    private lateinit var _view: View
    private var layout: CardView? = null
    private lateinit var _webView: WebView

    override val contentWebView: WebView
        get() = _webView

    override val view: View
        get() = _view

    override fun contentWebViewOnPageFinished() {}
    override fun contentWebViewOnReceivedError(description: String?) {}

    override fun getLayout(context: Context): Int {
        return R.layout.standard_display
    }

    override fun bindViews(v: View) {
        _view = v
        layout = v.findViewById(R.id.standard_display_layout)
        _webView = v.findViewById(R.id.standard_display_webview)
    }
}